server/api/opportunity/opportunity.js

```diff
@@ -30,7 +30,9 @@ const opportunitySchema = new Schema({
   requestor: { type: Schema.Types.ObjectId, ref: 'Person', required: true },
   dateAdded: { type: 'Date', default: Date.now, required: true },
   href: String,
-  tags: { type: [String], required: true, default: [] }
+  tags: { type: [String], required: true, default: [] },
+  organisationAddress: String,
```

Comment:
This should not be the organisationAddress.  If present it should be the address where the activity is taking place.  i.e locationAddress.  It should be initialised from the op.offer Org organisation address.